src/lib/packet.h

```diff
@@ -377,6 +377,7 @@ typedef enum {
                                * IETF-S/MIME) */
     PGP_PKA_EDDSA = 22,       /* EdDSA from draft-ietf-openpgp-rfc4880bis */
 
+    PGP_PKA_SM2_ENCRYPT = 98, /* SM2 encryption */
```

Comment:
I had a late question, are we sure it's best to add SM2 encryption as a separate alg? If SM2 can be used for both sign+encrypt+etc, we really only need one entry, right?

It would be similar to RSA, ignoring the deprecated 0x2 and 0x3, there's just one entry and it can be used for everything.